1-foundations/2-control-flows-and-functions/foundations-s2.ipynb

```diff
@@ -146,17 +156,33 @@
     "    print(country + \"'s GDP in 2020 was \"+ str(gdp_dict[country]) + \" billion US dollars.\")"
```

Comment:
I like using `.items()` and a great example of a method/function that returns a typle. If you want, maybe add as an example on how you can do the same thing in two different ways. But fully up to you.
```
for country, gdp in gdp_dict.items():
    print(country + "'s GDP in 2020 was "+ str(gdp) + " billion US dollars.")